packages/lime-proto-bmx7/files/etc/uci-defaults/bmx7-id-banner.defaults

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+echo " BMX7 ID: $(bmx7 -c show=status /r=1 | tail -n 1 | awk '{ print $2 }')" >> /etc/banner
```

Comment:
AFAIK when uci-defaults are executed the daemons (as bmx7) are not yet started. Is it possible to call "bmx7 -c" (connect) when the daemon is not running?

Reply:
Very right, I'll add it to `etc/profile`